tirocks-sys/bindings/bindings.rs

```diff
@@ -5428,3 +5428,54 @@ pub enum rocksdb_Histograms {
     DB_WRITE_WAL_TIME = 48,
     HISTOGRAM_ENUM_MAX = 49,
```

Comment:
rocksdb would always checking `HISTOGRAM_ENUM_MAX`, and tickers from Titan would be larger than it and cause panic